docs/user/adaptive-learning.rst

```diff
@@ -64,6 +67,35 @@ Alternatively, instructors can also link competencies to an exercise or lecture
 
 |instructors-learning-goals-link|
 
+Learning Paths
+--------------
+
+Learning Paths for Students
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Students can access their learning path in the learning path tab. Here they are able to access recommended lecture units and participate in exercises.
+Recommendations (visualized on the left) are generated via an intelligent agent that accounts for multiple metrics, e.g. prior performance, confidence, relations, and due dates, to support students in their selection of learning resources.
+Students can use the up and down buttons to navigate to the previous or next recommendation respectively. Hovering over a node in the list will display more information about the learning resource.
+
+|students-learning-path-participation|
+
+Students can access their learning path graph via the eye icon on the top left. The graph displays all competencies, lecture units, exercises, and their relations. Each competency consists of a start node, visualized by the competency rings displaying progress, confidence, and overall mastery, and an end node represented by a checkered flag. Learning resources are linked to a competency via edges tot he respective start and end nodes. If the resource is still pending, it is represented as a play symbol. Once the task is completed, it is visualized as a checkmark.
+Users can read the graph from top to bottom starting with the competencies that have no prerequisites, continuing downwards towards competencies that build up on the prior knowledge. Students can zoom, pan, and drag the graph to navigate. For better orientation, the top right corner contains a mini-map.
```

Comment:
"Users can read the graph from top to bottom starting with the competencies" -> "Users can read the graph from top to bottom, starting with the competencies"

"competencies that build _up on the_ prior knowledge" -> "competencies that build _upon/on_ prior knowledge"